Sources/MongoSwift/Collection.swift

```diff
@@ -701,6 +701,7 @@ public class MongoCollection<T: Codable> {
      * - Returns: The optional result of attempting to perform the insert. If the `WriteConcern`
```

Comment:
are there methods elsewhere we want to apply this? e.g. `Database.runCommand` maybe?

Reply:
added there - looking through the other API methods I think that's it, everything else is a read command so presumably you'd always want the results

Reply:
👍 LGTM